test/modules/MetricsValidator/src/tests/ValidateDocumentedMetrics.cs

```diff
@@ -97,6 +98,9 @@ async Task SeedMetrics(CancellationToken cancellationToken)
             await this.moduleClient.SetMethodHandlerAsync(methodName, (_, __) => Task.FromResult(new MethodResponse(200)), null);
             await this.moduleClient.InvokeMethodAsync(Environment.GetEnvironmentVariable("IOTEDGE_DEVICEID"), Environment.GetEnvironmentVariable("IOTEDGE_MODULEID"), new MethodRequest(methodName), cancellationToken);
 
+            await this.moduleClient.UpdateReportedPropertiesAsync(new TwinCollection(), cancellationToken);
+            await this.moduleClient.GetTwinAsync(cancellationToken);
```

Comment:
Why are you making these calls?

Reply:
There's a couple metrics like twin update duration and message latency that only show up if the relevant action has happened. This just does everything needed to populate all the metrics.

Reply:
Maybe a comment to that effect...